walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/script.js

```diff
@@ -13,8 +13,9 @@
 // limitations under the License.
 
 /** Creates a map and adds it to the page. */
+var map;
 function createMap() {
-  const map = new google.maps.Map(
+  map = new google.maps.Map(
       document.getElementById('map'),
       {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
make these variables

var defaultLat 37.422
lat: defaultLat

We don't want random numbers in our code w/o realizing why we have them